R/generate_nowcasts.R

```diff
@@ -0,0 +1,85 @@
+#' Generate retrospective nowcasts
+#'
+#' This function ingests a list of retrospective reporting triangles and
```

Comment:
I think I am a bit confused as to the precise role this plays which suggests to me it might need some more docs. 

It looks to me that it completes a triangle based on the delay in that triangle which is a special case of a nowcast? Do we therefore need to make that clear

Reply:
can you explain it shortly here as its a bit hard to review otherwise?